src/main/java/core/basesyntax/service/FileWriteImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriteImpl implements FileWrite {
+    @Override
+    public void write(String report, String fileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(fileName))) {
+            bufferedWriter.write(report);
+        } catch (IOException ioException) {
+            throw new RuntimeException("Data cannot write to file");
```

Comment:
According to the checklist, when rethrowing exceptions, you should provide an informative message and include the original exception object. Modify the catch block to include the path of the file that caused the IOException, and pass the original IOException object to the RuntimeException constructor.